src/main/java/com/smartbear/postman/collection/PostmanCollectionV2_1.java

```diff
@@ -0,0 +1,9 @@
+package com.smartbear.postman.collection;
+
+import net.sf.json.JSONObject;
+
+public class PostmanCollectionV2_1 extends PostmanCollectionV2 {
```

Comment:
This class extends nothing from the super class and thus it seems to be unneeded. Why this class was created?

Reply:
Yup, I think it may be removed for now and introduced once there are any differences in implementation :)